terraform/projects/infra-security-groups/main.tf

```diff
@@ -105,37 +105,6 @@ resource "aws_security_group" "monitoring_external_sg" {
   )}"
 }
 
-resource "aws_security_group" "alertmanager_external_sg" {
-  name        = "${var.stack_name}-alert-external-sg"
-  vpc_id      = "${data.terraform_remote_state.infra_networking.vpc_id}"
-  description = "Controls external access to the LBs"
-
-  tags = "${merge(
-    local.default_tags,
-    var.additional_tags,
-    map("Stackname", "${var.stack_name}"),
-    map("Name", "${var.stack_name}-monitoring-external-sg")
-  )}"
-}
-
-resource "aws_security_group_rule" "alertmanager_external_sg_ingress_office_http" {
-  type              = "ingress"
-  to_port           = 80
-  from_port         = 80
-  protocol          = "tcp"
-  security_group_id = "${aws_security_group.alertmanager_external_sg.id}"
-  cidr_blocks       = ["${var.cidr_admin_whitelist}"]
-}
-
-resource "aws_security_group_rule" "alertmanager_internal_alb" {
-  type                     = "ingress"
-  to_port                  = 80
-  from_port                = 80
-  protocol                 = "tcp"
-  security_group_id        = "${aws_security_group.alertmanager_external_sg.id}"
-  source_security_group_id = "${aws_security_group.monitoring_internal_sg.id}"
```

Comment:
This rule used to allow nginx and prometheus to access the internal ALB (in order to access prometheus and alertmanager for proxying/scraping/alerting).  What is the equivalent now that we've got rid of `alertmanager_external_sg`?